tests/scripts/docker/run_tests.sh

```diff
@@ -44,7 +44,7 @@ fi
 
 # CASS_DRIVER_NO_EXTENSIONS is set so we don't build the Cassandra C-extensions,
 # as this can take several minutes
-docker build --pull --force-rm --build-arg PYTHON_IMAGE=${1/-/:} -t apm-agent-python:${1} . # replace - with : to get the correct docker image
+docker build --force-rm --build-arg PYTHON_IMAGE=${1/-/:} -t apm-agent-python:${1} . # replace - with : to get the correct docker image
```

Comment:
The `--force-rm` will invalidate the packer cache

`--force-rm                Always remove intermediate containers`

```suggestion
docker build --build-arg PYTHON_IMAGE=${1/-/:} -t apm-agent-python:${1} . # replace - with : to get the correct docker image
```

Reply:
I did test this particular scenario:
- Given the packer_cache for the existing branch, run the packer_cache.sh script and the `docker build --force-rm --build-arg PYTHON_IMAGE=${1/-/:} -t apm-agent-python:${1} .` and it worked as expected, so it did use the cached docker images.

I also tried to use the cached docker images in the `master` branch by running `docker build --force-rm --build-arg PYTHON_IMAGE=${1/-/:} -t apm-agent-python:${1} .` and it worked as expected, so it did use the cached docker images.

Therefore it's not required the suggestion

Reply:
to be honest, I'm not 100% sure why `--force-rm` is there in the first place. I'd be fine removing it to see if anything breaks.